docs/pages/transactions/solana/sponsor-gas-solana.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Sponsor gas on Solana
+description: How to sponsor fees on Solana
+slug: wallets/transactions/solana/sponsor-gas
+---
+
+Let users send gasless transactions on Solana by sponsoring transaction fees and rent with Smart Wallets. This removes a major barrier to entry for new users of your app and the need for them to hold any SOL.
+
+## How it works
+
+Once you've [created](https://www.alchemy.com/docs/wallets/react/solana-wallets/get-started) a smart wallet on Solana, you can define what transactions you would like to sponsor using a gas manager policy and start sponsoring Solana fees. When a transaction meets the policy, fees and rent are fronted in SOL by Alchemy's fee payer so the user doesn’t need SOL. The equivalent USD amount and the admin fee are then added to your monthly invoice.
```

Comment:
i think we shouldn't use any full urls (including domain) that link to other docs.